src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -45,14 +45,14 @@
 						:aria-label="t('spreed', 'Share files to the conversation')"
 						:aria-haspopup="true">
 						<ActionButton
-							v-if="canShareAndUploadFiles"
+							v-if="canUploadFiles"
 							:close-after-click="true"
 							icon="icon-upload"
 							@click.prevent="clickImportInput">
 							{{ t('spreed', 'Upload new files') }}
 						</ActionButton>
 						<ActionButton
-							v-if="canShareAndUploadFiles"
+							v-if="canShareFiles"
 							:close-after-click="true"
 							icon="icon-folder"
```

Comment:
this gets hover feedback when the mouse is in the chat view, while the other other one does not
![Peek 2021-01-29 10-25](https://user-images.githubusercontent.com/213943/106257010-50d9f600-621c-11eb-80f9-e571e2fd0b8b.gif)

Reply:
There's this problem also with the topbar buttons, it was introduced recently. I don't think it's related to this pr

Reply:
ok

Reply:
I'm seeing slightly different symptoms:

1. hovering outside there is no circle at all
2. hovering on the message list (when empty), the circle appears light grey
3. hovering on the button makes the circle dark grey

Anyway, raised as https://github.com/nextcloud/spreed/issues/5049